corehq/apps/reports/views.py

```diff
@@ -646,7 +651,8 @@ def recalculate_hour(hour, hour_difference, minute_difference):
 
 
 def get_timezone_difference(domain):
-    return datetime.now(pytz.timezone(Domain.get_by_name(domain)['default_timezone'])).strftime('%z')
+    tz_diff = datetime.now(pytz.timezone(Domain.get_by_name(domain)['default_timezone'])).strftime('%z')
+    return Timezone(tz_diff[:3], tz_diff[3:])
```

Comment:
you could push the `int` conversion in here as well instead of having to repeat it in many places
```suggestion
    return Timezone(int(tz_diff[:3]), int(tz_diff[3:]))
```

Reply:
Ah, totally forgot about those!